mpt_api_client/resources/commerce/mixins.py

```diff
@@ -0,0 +1,186 @@
+import httpx
+
+from mpt_api_client.models import ResourceData
+
+
+class ValidateMixin[BaseModel]:
```

Comment:
One question... Do you really need all of the these action-based mixins here? I mean, it make sense when you are trying to share some functionality, like update/create/etc. But for specific actions for orders - what's the reason to have it as mixin per cada action?